### ISSUE TITLE
HW02代码是不是已经写错了schduler的入参？

### ISSUE THREAD

**Issue by lsycxyj** [Hoper-J/HUNG-YI_LEE_Machine-Learning_Homework#1]
原文
```python
    scheduler.step(best_acc/len(val_set))
```
应该是
```python
    scheduler.step(val_loss/len(val_set))
```
吧？

**Comment by Hoper-J** [comment]
是的，应该传入 scheduler.step(val_acc/len(val_set))，非常感谢指正，现在的入参会让它过早的降低 lr。

**Comment by Hoper-J** [comment]
不过 HW02 的结果没有影响，因为仅 train 了 15 个 epoch。

> 是的，应该传入 scheduler.step(val_acc/len(val_set))，非常感谢指正，现在的入参会让它过早的降低 lr。
